handbook/engineering/code-intelligence/code-walkthrough.md

```diff
@@ -0,0 +1,57 @@
+# Code walkthrough
+
+The following page details how we approach the technical hiring within the Code Intelligence team instead of take home projects.
+
+### What is the day to day engineering experience at Sourcegraph?
+- We're often reading code we haven't written.
+- We're functioning in an asynchronous environment where clear communication is key.
+- We need to help our team understand areas where we are the domain expert.
+
+### What do we want to be true about this interview?
+- Allow you as the candidate to focus on an area that you are comfortable in.
+- Allow a dialogue that simulates a technical discussion between your potential future peers.
+- Use the discussion as a proxy for technical aptitude and programming skills.
+- Have a time efficient process for yourself and Sourcegraph.
+
+### We want to find engineers that are:
+- Good communicators.
+- Can tailor the technical depth and scope of a conversation dependent on the audience.
+- Can effectively teach and share their knowledge.
+
+----
+
+### Overview
+
+For this interview you'll drive a walkthrough of either a personal project or library of your choice.
+
+The following are sample ideas, the most important thing is you choose a library that you are comfortable and familiar with.  Walkthroughs can be done in
+any language, we do have a preference if possible that they are close to the domain you'd be working in.
+
+- For a full-stack position you may choose a library that you've used heavily such as [DateJs](https://github.com/datejs/Datejs)
```

Comment:
I feel the phrase "such as" might be mistaken as meaning only one of these two, and would probably prefer the phrase "for example"

Reply:
Do you think it's better to qualify both with a sentence above like:

_The following libraries are indicated as examples only_ ?

Reply:
I think an explicit "for example" instead of "such as" would make it clearer that these are examples.

> For a full-stack position you may choose a library that you've used heavily. For example, DateJs or Redux.

Reply:
Will call this out clearer!

Reply:
New phrasing of ```- For a full-stack position you may choose a library that you've used heavily, for example [DateJs](https://github.com/datejs/Datejs)
or [Redux](https://github.com/reduxjs/redux).```

Happy to keep iterating because I *definitely* don't want candidates to be stuck with ambiguity.